content/en/docs/overview/why-choose-istio/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Why choose Istio?
+description: Compare Istio to other service mesh solutions.
+weight: 20
+keywords: [comparison]
+owner: istio/wg-docs-maintainers-english
+test: n/a
+---
+
+Istio pioneered the concept of a sidecar-based service mesh when it launched in 2017. Out of the gate, the project included the features that would come to define a service mesh, including standards-based mutual TLS for zero-trust networking, smart traffic routing, and observability through metrics, logs and tracing.
+
+Since then, the project has driven advances in the mesh space including [multi-cluster & multi-network topologies](/docs/ops/deployment/deployment-models/), [extensibility via WebAssembly](/docs/concepts/wasm/), the [development of the Kubernetes Gateway API](/blog/2022/gateway-api-beta/), and moving the mesh infrastructure away from application developers with [ambient mode](/docs/ambient/overview/).
+
+Here are a few reasons we think you should use Istio as your service mesh.
+
+## Simple and powerful
+
+Kubernetes has hundreds of features and dozens of APIs, but you can get started with it with just one command. We've built Istio to be the same way. Progressive disclosure means you can use a small set of APIs, and only turn the more powerful knobs if you have the need. Other "simple" service meshes spent years catching up to the feature set Istio had on day 1.
+
+It is better to have a feature and not need it, than to need it and not have it!
+
+## The Envoy proxy {#envoy}
```

Comment:
Should we mention about the proxy usage of ambient mode as well?

Reply:
We do in the "Why not use a per-node proxy?" section.  I've added another sentence to clarify here.